appservice/src/createAppService/AppServicePlanListStep.ts

```diff
@@ -64,11 +69,13 @@ export class AppServicePlanListStep extends AzureWizardPromptStep<IAppServiceWiz
     }
 
     private async getQuickPicks(wizardContext: IAppServiceWizardContext): Promise<IAzureQuickPickItem<WebSiteManagementModels.AppServicePlan | undefined>[]> {
-        const picks: IAzureQuickPickItem<WebSiteManagementModels.AppServicePlan | undefined>[] = [{
-            label: localize('CreateNewAppServicePlan', '$(plus) Create new App Service plan'),
-            description: '',
-            data: undefined
-        }];
+        const picks: IAzureQuickPickItem<WebSiteManagementModels.AppServicePlan | undefined>[] = !wizardContext.disablePlanCreate
```

Comment:
Could you please follow this design pattern if you need to remove the `Create new App Service plan` quickpick?

https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/wizard/ResourceGroupListStep.ts#L23-L29
https://github.com/microsoft/vscode-azuretools/blob/main/ui/src/wizard/ResourceGroupListStep.ts#L70-L79

Reply:
Updating with class variable.